### DISCUSSION TITLE
Share experience and coordinate distros

### DISCUSSION THREAD

**Discussion by Maxzor** [rocm-arch/rocm-arch#674] [General]
Dear Arch ROCm maintainers,
There are on-going talks between AMD and Debian to try to improve a bit the cmake.
https://lists.debian.org/debian-ai/2022/02/msg00046.html
You are more than welcome to chime in and ensure your views are taken into account!
BR, Maxime

**Comment by acxz** [comment]
Whoa this is amazing! Thanks for keeping us in the loop and noticing us @Maxzor . Better CMake integration will definitely make our lives easier here for packaging on Arch. I'll definitely be following that thread.

**Comment by tpkessler** [comment]
Hey @Maxzor, thanks for pointing us to this thread! We have no direct contact to ROCm devs but @acxz tried to add a section on Arch to the official docu, 

https://github.com/RadeonOpenCompute/ROCm_Documentation/pull/73

It eventually became part of an FAQ

https://community.amd.com/t5/server-gurus-discussions/faq-installation-instructions-for-archlinux/td-p/316662

**Comment by Maxzor** [comment]
You are welcome, you were a very early adopter of ROCm as a distro!

I try to push for the FHS and cmake GNUInstallDirs, Jeremy Newton AMD dev and Fedora packager, and Cory Bloor, AMD dev are quite on the same page... but no clue if this will ever give birth to anything.
https://lists.debian.org/debian-ai/2022/02/msg00065.html
